docs/manual-CN/3.build-develop-and-administration/2.install/1.install-with-rpm-deb.md

```diff
@@ -87,20 +84,6 @@ sudo dpkg -i nebula-2019.12.23-nightly.ubuntu1604.amd64.deb
 rpm -ivh --prefix=${your_dir} nebula-graph-${version}.rpm
 ```
 
-* 如需将 **Nebula Graph** 打包至一个包，请使用以下命令：
-
-```bash
-cd nebula/package
-./package.sh -v <version>
-```
-
-* 如需将 **Nebula Graph** 打包至多个包，请使用以下命令：
-
-```bash
-cd nebula/package
-./package.sh -v <version> -n OFF
-```
-
```

Comment:
Please place them in the `build source code` chapter.

Reply:
I'll fix in a new pr.